homeassistant/config_entries.py

```diff
@@ -706,3 +717,20 @@ def __init__(self, hass: HomeAssistant) -> None:
 
         result["result"] = True
         return result
+
+
+@attr.s(slots=True, frozen=True)
```

Comment:
Can't we mark it as non-frozen, and just assign all properties in the `data` dict as properties ?